app/views/handlers/feedback.py

```diff
@@ -111,6 +115,18 @@ def get_page_title(self):
         return self.PAGE_TITLE
 
     def handle_post(self):
+        feedback_data = {
+            "feedback_text": self.form.data["feedback-type"],
+            "feedback_topic": self.form.data["feedback-text"],
+        }
+
+        feedback_upload = current_app.eq["feedback"].upload(
+            self._schema, feedback_data, self._session_store.session_data
+        )
```

Comment:
As specified on the card, it was agreed that add all non form data attributes should be included as metadata and payload. Therefore, this could be simplified by passing 2 things into the method, `data` and `metadata` where metadata is a subset of data without "`feedback_text` and `feedback_topic`.